packages/eclwatch/src/WUResultStore.ts

```diff
@@ -182,11 +185,25 @@ export class Store {
                     return formattedRow;
                 })
             };
+        }).catch((err: Exception) => {
+            this._lastError = err.Message || "An exception has occurred";
+            if (this.onError) {
+                this.onError(this._lastError);
+            }
+            return { totalLength: 0, data: [] };
         });
         this._cache[cacheKey] = retVal;
         return retVal;
     }
 
+    hasError() {
+        return !!this._lastError;
+    }
+
+    getLastError() {
+        return this._lastError;
+    }
```

Comment:
This is probably not needed?

If they are needed, then this._lastError should get cleared prior to fetch call.